src/app/payment/PaymentSubmitButton.tsx

```diff
@@ -55,7 +55,7 @@ const PaymentSubmitButtonText: FunctionComponent<PaymentSubmitButtonTextProps> =
     }
 
     if (methodType === PaymentMethodType.Paypal) {
-        return <TranslatedString id="payment.paypal_continue_action" />;
+        return <TranslatedString id={ methodId === 'braintreevenmo' ? 'payment.braintreevenmo_continue_action' : 'payment.paypal_continue_action' } />;
```

Comment:
Instead of using `'braintreevenmo'` we should add this to the `PaymentMethodId` enum and reference it from there.

Reply:
fixed